components/link/stories/link.stories.js

```diff
@@ -35,7 +36,7 @@ export default {
 				type: { summary: "string" },
 				category: "Component",
 			},
-			options: ["secondary"],
+			options: ["default/primary", "secondary"],
```

Comment:
Instead of including default in the dropdown name, you can leave this as primary and indicate that primary is the defaultValue: https://storybook.js.org/docs/api/arg-types#tabledefaultvalue

Reply:
Love it: https://github.com/adobe/spectrum-css/pull/2884/commits/11991c12176d65f696c58b019c7d289b13826888